internal/httputil/httputil_z_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright GoFrame Author(https://goframe.org). All Rights Reserved.
+//
+// This Source Code Form is subject to the terms of the MIT License.
+// If a copy of the MIT was not distributed with this file,
+// You can obtain one at https://github.com/gogf/gf.
+
+package httputil
+
+import (
+	"net/url"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Please use `gtest` for simple testing cases, not import `testify` into goframe, referring to other ut cases for example.

Add the ut case to file `ghttp_z_unit_test.go`, not creating a new ut file.